notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # Declare dictionary with keys column names and values User Defined Functions (UDF) and return types.
</pre><p>Do you introduce these in a non-code cell at all? Or is it only inline comments? given that this is where the majority of the fun stuff happens, is it worth having a few cells to show what's under the hood? Defer to Brad on this for a final decision if you're not sure</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>